_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,223 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each [`<meta>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element) element with a [`name`](https://www.w3.org/TR/html52/document-metadata.html#element-attrdef-meta-name) attribute whose value is set to [`viewport`](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Enter_viewport_meta_tag) and has a [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute.
```

Comment:
Are `viewport`, `user-scalable` and `maximum-scale` case-sensitive?
The way the rule is currently written, they are. But does that reflects reality?

Reply:
Yup I think so - https://www.w3.org/TR/css-device-adapt-1/#meta-properties.

Reply:
* Names (_i.e._ `viewport`) seem to be case-insensitive: https://html.spec.whatwg.org/multipage/semantics.html#standard-metadata-names

* I'm having harder time figuring if properties names and values (_i.e._ what's inside `content`) are case sensitive or not. The link you put list them without explicitly mentioning case-sensitivity (I agree that I would assume it by default). It seems that Chrome does not respect meta viewport and I do not have a Safari to test, can you run some tests?

Reply:
@Jym77 as discussed on a call, I tested various casing locally & it looks like meta tags and their values are case insensitive.

Some relevant links:
- https://support.google.com/webmasters/answer/79812?hl=en
- https://webmasters.stackexchange.com/questions/67760/which-meta-names-are-case-sensitive
- https://html.spec.whatwg.org/multipage/dom.html#case-insensitivity

Reply:
Then, we should put case-insensitivity in the rule, _e.g._ "witha `name` attribute is a case-insensitive match for `viewport`" and so on.